TombLib/TombLib/LevelData/Compilers/TombEngine/PathfindingDecompiled.cs

```diff
@@ -32,10 +32,10 @@ public sealed partial class LevelCompilerTombEngine
         private bool dec_flipped;
         private bool dec_jump;
         private Room dec_currentRoom;
-        private short dec_q0 = -1;
-        private short dec_q1 = -1;
-        private short dec_q2 = -1;
-        private short dec_q3 = -1;
+        private int dec_q0 = Clicks.ToWorld(-1);
```

Comment:
I don't understand this. Why these 4 variables were converted to full world unit scale, if all other underlying code remains the same?

Reply:
Because `dec_q0` now stores world units instead of clicks (at least for TEN).
Honestly, Pathfinding code is a mystery to me, because it seems that it doesn't actually go anywhere most of the time.
Also keeping it in Clicks made the AI go wild originally, but maybe it was something else. I wouldn't risk changing it back though.

Reply:
You mean that AI now behaves better than before this change? If so, it may be another incorrect decompilation which got fixed only now.